### PR TITLE
Multiple improvements to prepare for rascaline-torch moving day

### PR DESCRIPTION
**Pull request by Luthaf**
This PR contains a handful of small changes/cleanup unrelated to rascaline-torch but that should happen before the code moves to it's new home.

----
:books: Documentation preview :books:: https://rascaline--199.org.readthedocs.build/en/199/

### CODE REVIEW

File: rascaline-c-api/include/rascaline.hpp [hunk 1]
```diff
@@ -606,28 +606,62 @@ class Calculator {
...
+            1,
             options.as_rascal_calculation_options_t()
         ));
 
         return equistore::TensorMap(descriptor);
     }
 
-    /// Runs a calculation with this calculator on the given basic ``systems``
-    equistore::TensorMap compute(BasicSystems& systems, const CalculationOptions& options = CalculationOptions()) const {
+    /// Runs a calculation for all the `systems` read from a file
```
Comment (PicoCentauri) [inline rascaline-c-api/include/rascaline.hpp:660]: There is no file reading here.
```suggestion
    /// Runs a calculation for all the `systems`
```

**Review by PicoCentauri — COMMENTED** [review]
Very useful. Just a minor docs comment.

File: rascaline-c-api/include/rascaline.hpp (hunk 1 above)
Comment (Luthaf) [reply rascaline-c-api/include/rascaline.hpp:660]: `BasicSystems` is a list of system that where created from a file. This is a bit unclear, I'll try to improve it!

Comment (Luthaf) [reply rascaline-c-api/include/rascaline.hpp]: This is a convenience function for the C++ interface, to not have to create a custom `System` all the time

**Review by PicoCentauri — APPROVED** [review]
Okay better docstring. From the parameters it was not clear that this contains a file constructor.

**Pull request merged**